src/applications/ivc-champva/10-7959C/chapters/formSignature.js

```diff
@@ -9,6 +11,21 @@ export const formSignatureSchema = {
     ...titleUI('Your information'),
     certifierRole: {
       ...radioUI({
+        updateUiSchema: formData => {
+          const labels = {
+            applicant: `I'm filling out this form for myself`,
+            other: `I'm a parent, spouse, or legal representative signing on behalf of ${
+              formData?.applicantName?.first
+            }`,
+          };
+
+          return {
+            'ui:title': `Which of these best describes you?`,
+            'ui:options': {
+              labels,
+            },
+          };
+        },
```

Comment:
I believe this function should look a little different as of [this recent PR](https://github.com/department-of-veterans-affairs/vets-website/pull/33388/files#diff-b07cb510c6203f7cec2d4a2a6a07960dc5079f7e9c6c9875eec81558cf94cc77) - we should be able to get rid of the `updateUiSchema` function here and just have the label keys